MiniAn/minian_main.py

```diff
@@ -467,6 +471,115 @@ def load_doric_to_xarray(
 
     return varr, file_
 
+            
+def cross_register(AC, A, minian_parameters):
+    perform_cross_reg = minian_parameters.params_cross_reg["cross_reg"]
+    if not perform_cross_reg:
+        return A
+
+    print(mn_defs.Messages.CROSS_REGISTRATING , flush=True)
+    param_dist = 5 # Defines the max dist between centroids on different sessions to consider them as same cell
+    ref_filename = minian_parameters.params_cross_reg["fname"]
+    ref_images   = minian_parameters.params_cross_reg["h5path_images"]
+    ref_ROIs     = minian_parameters.params_cross_reg["h5path_roi"]
+
+    # Load base/reference file to xarray
+    ref, file_ref =  load_doric_to_xarray(ref_filename, ref_images, np.float64, None, "subset", None)
+
+    # max project of the miniAn images for base file and current file
+    ref_max     = ref.max("frame")
+    AC_max = AC.max("frame")
+
+    concatenated_maxAC = xr.concat([ref_max, AC_max], pd.Index(["session1", "session2"], name = "session"))
+
+    # Estimate a translational shift along the session dimension using the max projection for each dataset.
+    shifts = estimate_motion(concatenated_maxAC, dim = "session").compute().rename("shifts")
+    # Apply Shifts
+    transformed = apply_transform(concatenated_maxAC, shifts).compute().rename("temps_shifted")
+    shiftds = xr.merge([concatenated_maxAC, shifts, transformed])
+
+    window    = shiftds["temps_shifted"].isnull().sum("session")
```

Comment:
These 2 lines are part of the "visualize overlap of field of view across all sessions" of the cross-registration.ipynb. Do we need them here? The `window` is calculated on the line 524, isn't it?